File: backend/src/main/java/com/revnu/backend/features/settings/dto/RestaurantSetupRequest.java
package com.revnu.backend.features.settings.dto;

import java.time.LocalTime;
import com.fasterxml.jackson.annotation.JsonFormat;

public record RestaurantSetupRequest(
        String name,
        String physicalLocation,
        @JsonFormat(pattern = "HH:mm")
        LocalTime openingHrs,
        @JsonFormat(pattern = "HH:mm")
        LocalTime closingHrs
        ) {

}

File: mobile/app/src/main/java/com/revnu/mobile/features/restaurant/model/RestaurantSetupRequest.kt
package com.revnu.mobile.features.restaurant.model

import com.google.gson.annotations.SerializedName

data class RestaurantSetupRequest(
    @SerializedName("name")
    val name: String,

    @SerializedName("physicalLocation")
    val physicalLocation: String,

    @SerializedName("openingHrs")
    val openingHrs: String,

    @SerializedName("closingHrs")
    val closingHrs: String
)